shtdlib.sh

```diff
@@ -871,7 +907,7 @@ function copy_dir {
     elif [ -n "$(find ${find_directory} -maxdepth 1 -name ${find_pattern} -type f -print -quit)" ] ; then
         debug 10 "Found globbing pattern in ${source}"
         set_dir_perm_owner "${source}" "${owner_group}" "${file_perm}" "${dir_perm}"
-        if ${force_overwrite} ; then
+        if ${force_overwrite:-false} ; then
```

Comment:
Ditto.

Reply:
I don't think so, since true/false are actual commands and if uses their return status (0 or 1) adding quotes would make if try to interpret the string right ?